src/ui/AvatarSet.ts

```diff
@@ -37,6 +37,9 @@ export default class AvatarSet {
             let element = $(this);
 
             element.css('background-image', `url(${avatar.getData()})`);
+            element.css('background-size', 'contain');
+            element.css('background-repeat', 'no-repeat');
+            element.css('background-position', 'center');
```

Comment:
Should be handled by css. Did you have issues with that?

Reply:
i had issues in chrome... chrome complained about the dataurl as invalid property when there was no background-size given...